cmd/core-common-config-bootstrapper/entrypoint.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/dumb-init /bin/sh
+#  ----------------------------------------------------------------------------------
+#  Copyright (c) 2023 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+#  SPDX-License-Identifier: Apache-2.0
+#  ----------------------------------------------------------------------------------
+
+set -e
+
+$@ && exec tail -f /dev/null
```

Comment:
```suggestion
"$@" && exec tail -f /dev/null
```

See https://stackoverflow.com/questions/1668649/how-to-keep-quotes-in-bash-arguments

Basically, "$@" is special syntax to make sure that the input arguments to the script are not re-split on whitespace.  It is the equivalent of "$1" "$2" "$3", et cetera.   This is a SPECIAL CASE and breaks all the normal bash rules.